src/securejoin.rs

```diff
@@ -1170,7 +1170,10 @@ mod tests {
                 } => {
                     let alice_contact_id =
                         Contact::lookup_id_by_addr(&bob.ctx, "alice@example.com", Origin::Unknown)
-                            .await;
+                            .await
+                            .ok()
```

Comment:
Why not just `.unwrap().unwrap()` instead of `.ok().flatten().unwrap()`? Or, even better, `.expect("Error looking up the contact").expect("Contact not found")`

Reply:
I thought `ok().flatten().unwrap()` made it more clear you have a result of an option.  But the `.exepect()` calls are even better I guess because it doesn't really matter whether they're options or results in that case.  Good call.